doc/netplan-tutorial.md

```diff
@@ -225,518 +244,214 @@ network:
+New or modified Netplan settings need to be applied before they take effect on a running system.
 
-Use your favourite text editor and open the file `/etc/netplan/second-interface.yaml`.
+:::{note}
+Using the `netplan set` command to modify configuration or editing (creating) the Netplan YAML configuration files directly does not automatically apply the new settings to the running system.
+:::
 
-Add the configuration below to the interface configuration section:
+After creating a new configuration as described in [Using netplan set to enable a network interface with DHCP](#using-netplan-set-to-enable-a-network-interface-with-dhcp), follow these steps to apply the settings and confirm they have taken effect.
 
-```yaml
-accept-ra: false
-link-local: []
-```
+1. Display the current state of the network interface:
 
-When you finish, it should look like this:
+   ```
+   ip address show enp6s0
+   ```
 
-```yaml
-network:
-  version: 2
-  ethernets:
-    enp6s0:
-      dhcp4: true
-      accept-ra: false
-      link-local: []
-```
+   Where `enp6s0` is the interface you wish to display status for. You should see an output similar to the one below:
 
-With this new configuration, the network configuration back end (systemd-networkd in this case) will not accept Route Advertisements and will not add the link-local address to our interface.
+   ```none
+   3: enp6s0: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
+       link/ether 00:16:3e:0c:97:8a brd ff:ff:ff:ff:ff:ff
+   ```
 
-Now check your new configuration with the `netplan get` command:
+   This interface has no IP address and its state is `DOWN`.
 
-```
-netplan get
-```
+2. Apply the new Netplan configuration:
 
-You should see something similar to this:
+   ```
+   netplan apply
+   ```
 
-```yaml
-network:
-  version: 2
-  ethernets:
-    enp5s0:
-      dhcp4: true
-    enp6s0:
-      dhcp4: true
-      accept-ra: false
-      link-local: []
-```
+3. Check the state of the `enp6s0` interface again using one of the following two methods:
 
-Now use `netplan apply` to apply your new configuration:
+   * Using the `ip` tool:
 
-```
-netplan apply
-```
+     ```
+     ip address show enp6s0
+     ```
 
-And check your interface configuration:
+     You should see an output similar to this:
 
-```
-ip address show enp6s0
-```
+     ```none
+     3: enp6s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
+         link/ether 00:16:3e:0c:97:8a brd ff:ff:ff:ff:ff:ff
+         inet 10.33.59.157/24 metric 100 brd 10.33.59.255 scope global dynamic enp6s0
+           valid_lft 3589sec preferred_lft 3589sec
+         inet6 fd42:ee65:61d0:abcb:216:3eff:fe0c:978a/64 scope global dynamic mngtmpaddr noprefixroute
+           valid_lft 3599sec preferred_lft 3599sec
+         inet6 fe80::216:3eff:fe0c:978a/64 scope link
+           valid_lft forever preferred_lft forever
```

Comment:
I see this can be a bit confusing.

The intended flow is:

1. Checking existing config.
2. Modifying existing or creating new config.
    * Using `netplan set`.
    * Using manual editing/creation of YAML config. files.
3. Applying new config.

For step 2, I opted not to show the exact same thing (i.e. making the same changes to config both using `netplan set` and manual editing), so that it wouldn't repetitive.

Therefore, in step 3, I had to choose which new config would be shown as being applied. I chose the new config created using `netplan set`, and I explained that in the beginning of the section (_After creating a new configuration as described in **Using `netplan set` to enable a network interface with DHCP**, follow these steps to apply the settings and confirm they have taken effect._).

So, on the one hand, there's this potential confusion (why is there IPv6 when we showed how to get rid of it?), but I think it also makes sense because, theoretically, the user would pick one of the two options in step 2 -- not both.

I can change it if you think it doesn't make sense or is just confusing.

Reply:
Yes. I really like how the sections build upon each other, showing different ways of doing things.

But many people might just work through the full tutorial linearly, doing both sections. Which is possible and totally valid. Then they might wonder about missing IPv6. If people do not work on the second "manual YAML edit" section, they might not care about IPv6 being there or not.

So I'd prefer to have it the other way around, considering both sections to be executed. It should not show IPv6 addresses.